types/smartystreets-javascript-sdk/index.d.ts

```diff
@@ -0,0 +1,379 @@
+        buildInternationalStreetClient(): Client;
+    }
+
+    namespace buildClient {
+        function usStreet(credentials: StaticCredentials | SharedCredentials): Client;
+        function usAutocomplete(
+            credentials: StaticCredentials | SharedCredentials,
+        ): Client;
+        function usAutocompletePro(
+            credentials: StaticCredentials | SharedCredentials,
+        ): Client;
+        function usExtract(credentials: StaticCredentials | SharedCredentials): Client;
+        function usZipcode(credentials: StaticCredentials | SharedCredentials): Client;
+        function internationalStreet(
+            credentials: StaticCredentials | SharedCredentials,
+        ): Client;
+    }
+
+    namespace Errors {
+        class SmartyError extends Error {
+            constructor(message: string);
+        }
+        class BatchFullError extends SmartyError {
+            constructor();
+        }
+        class BatchEmptyError extends SmartyError  {
+            constructor();
+        }
+        class UndefinedLookupError extends SmartyError  {
+            constructor();
+        }
+       class  BadCredentialsError extends SmartyError  {
+            constructor();
+        }
+        class PaymentRequiredError extends SmartyError  {
+            constructor();
+        }
+        class RequestEntityTooLargeError extends SmartyError  {
+            constructor();
+        }
+        class BadRequestError extends SmartyError  {
+            constructor();
+        }
+        class UnprocessableEntityError extends SmartyError  {
+            constructor(message: string);
+        }
+        class TooManyRequestsError extends SmartyError  {
+            constructor();
+        }
+        class InternalServerError extends SmartyError  {
+            constructor();
+        }
+        class ServiceUnavailableError extends SmartyError  {
+            constructor();
+        }
+        class GatewayTimeoutError extends SmartyError  {
+            constructor();
+        }
+    }
+    type Lookup = usStreet.Lookup | usZipcode.Lookup | usAutocomplete.Lookup | usAutocompletePro.Lookup | usExtract.Lookup | internationalStreet.Lookup;
+}
+
+export namespace usStreet {
+    class Lookup {
+        constructor(street1?: string, street2?: string, secondary?: string, city?: string, state?: string, zipCode?: string,
+            lastLine?: string, addressee?: string, urbanization?: string, match?: string, maxCandidates?: string,
+            inputId?: string);
+        street: string;
+        street2: string;
+        secondary: string;
+        city: string;
+        state: string;
+        zipCode: string;
+        lastLine: string;
+        adressee: string;
+        urbanization: string;
+        match: string;
+        maxCandidates: number;
+        inputId: string;
+        result: [Candidate];
+    }
+    class Candidate {
+        constructor(responseData: any);
+        inputIndex: any;
+		candidateIndex: any;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/smartystreets-javascript-sdk/index.d.ts;
  Line: 159;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

Reply:
switched tabs to spaces